en/uavcan/ark_rtk_gps.md

```diff
@@ -80,9 +82,41 @@ General instructions for UAVCAN PX4 configuration can also be found in [UAVCAN >
 ### PX4 Configuration
 
 You need to set necessary [UAVCAN](README.md) parameters and define offsets if the sensor is not centred within the vehicle.
+- Enable GPS yaw fusion by setting bit 7 of [EKF2_AID_MASK](../advanced_config/parameter_reference.md#EKF2_AID_MASK) to true.
 - Enable [UAVCAN_SUB_GPS](../advanced_config/parameter_reference.md#UAVCAN_SUB_GPS), [UAVCAN_SUB_MAG](../advanced_config/parameter_reference.md#UAVCAN_SUB_MAG), and [UAVCAN_SUB_BARO](../advanced_config/parameter_reference.md#UAVCAN_SUB_BARO).
+- Set [CANNODE_TERM](../advanced_config/parameter_reference.md#CANNODE_TERM) to `1` if this is that last node on the CAN bus.
 - The parameters [EKF2_GPS_POS_X](../advanced_config/parameter_reference.md#EKF2_GPS_POS_X), [EKF2_GPS_POS_Y](../advanced_config/parameter_reference.md#EKF2_GPS_POS_Y) and [EKF2_GPS_POS_Z](../advanced_config/parameter_reference.md#EKF2_GPS_POS_Z) can be set to account for the offset of the ARK RTK GPS from the vehicles centre of gravity.
 
+### Setting Up Moving Baseline & GPS Heading
+
+The simplest way to set up moving baseline and GPS heading with two ARK RTK GPS modules is via CAN, though it can be done via UART to reduce traffic on the CAN bus if desired.
+
+Setup via CAN:
+- Ensure the ARK RTK GPS modules are connected to the Pixhawk via CAN (one can connect to another's secondary CAN port)
+- Choose one ARK RTK GPS to be the *Rover* and one to be the *Moving Base*
+- Reopen QGroundControl, go to parameters, and select `Standard` to hide that dropdown and select `Component ##` to view each of your ARK RTK GPS's CAN node parameters
+:::note
+`Component ##` won't be visible unless the ARK RTK GPS is connected to the Pixhawk prior to opening QGroundControl
+:::
+- On the *Rover*, set the following:
+  - [GPS_UBX_MODE](../advanced_config/parameter_reference.md#GPS_UBX_MODE) to `3`
+  - [GPS_YAW_OFFSET](../advanced_config/parameter_reference.md#GPS_YAW_OFFSET) to `0` if your *Rover* is in front of your *Moving Base*, `90` if *Rover* is right of *Moving Base*, `180` if *Rover* is behind *Moving Base*, or `270` if *Rover* is left of *Moving Base*
+- On the *Moving Base*, set the following:
+  - [GPS_UBX_MODE](../advanced_config/parameter_reference.md#GPS_UBX_MODE) to `4`
+
+Setup via UART:
+- Ensure the ARK RTK GPS modules are connected to the Pixhawk via CAN
```

Comment:
Do you mean this? i.e. is CAN connection required for the UART setup. I expect so, but just checking

Reply:
@AlexKlimaj , I believe so right?

Reply:
Cool. I'll merge anyway and we can update if needed. Thanks for the quick turnaround.